site/navigation.js

```diff
@@ -29,7 +26,12 @@ class Navigation extends React.Component {
               className={linkClasses}
               href={member.route}
             >
+              {level === 0 ? <svg
+                style={{ marginLeft: '-18px' }}
+                className='absolute left icon'
+                dangerouslySetInnerHTML={{ __html: '<use xlink:href="#icon-caret-right"></use>' }} /> : ''}
```

Comment:
Breaking up the nav like this is a great improvement, but the caret definitely reads as an expandable dropdown to me. Can you try experimenting with a `ul` or a dash or something?